src/main/java/com/faforever/api/user/UserService.java

```diff
@@ -25,23 +27,25 @@
 @Slf4j
 public class UserService {
 
-  private static final Pattern USERNAME_PATTERN = Pattern.compile("[A-Za-z][A-Za-z0-9_-]{2,15}$");
   static final String KEY_ACTION = "action";
   static final String KEY_EXPIRY = "expiry";
   static final String KEY_USERNAME = "username";
   static final String KEY_EMAIL = "email";
   static final String KEY_PASSWORD = "password";
+  static final String KEY_USER_ID = "id";
```

Comment:
What you mean with dynamic? 
Why is another class a problem? 
It should reduce the code and make it more readable
https://github.com/FAForever/faf-java-api/pull/70